app/javascript/components/student/editor/TestsList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { TestSummary } from './TestSummary'
+import { TestStatus, Test } from './TestRunSummary'
+
+function Overview({ tests }: { tests: Test[] }) {
+  const passed = tests.reduce(function (total, test) {
+    if (test.status === TestStatus.PASS) {
+      total += 1
+    }
+
+    return total
+  }, 0)
+  const failed = passed === tests.length ? 0 : 1
```

Comment:
Assumes fail-fast. That's fine, but we should document it as a comment. If this changes in the future, this must change.

Also don't know if it's true for all tracks.

Reply:
This is for the purposes of our test display, and isn't dependent on any track. I think it is fine like this as this is really our assumption. I'm wary to add a comment as it will be harder to keep that comment up to date.